modules/shapefile/test/dbf/parse-dbf.spec.js

```diff
@@ -0,0 +1,41 @@
+import test from 'tape-promise/tape';
+import {fetchFile, load} from '@loaders.gl/core';
+import {DBFLoader} from '@loaders.gl/shapefile';
+
+const SHAPEFILE_JS_DATA_FOLDER = '@loaders.gl/shapefile/test/data/shapefile-js';
+const SHAPEFILE_JS_TEST_FILES = [
+  'boolean-property',
+  'date-property',
+  // 'latin1-property', // fails on 'México'
```

Comment:
Why?

Reply:
Right now all tests are run using `utf8` encoding, and as of last night I didn't yet fix the encoding polyfills for Node 12. (I believe when I uncomment that line the test passes on Node 14).

Reply:
I see, yes since you are testing DBF in isolation you don't have the code page.